src/document-click.js

```diff
@@ -68,40 +72,68 @@ export class ClickHandler {
     /** @private @const {!./service/history-impl.History} */
     this.history_ = historyFor(this.win);
 
+    /** @private @const {!./service/url-replacements-impl.UrlReplacements} */
+    this.urlReplacements_ = urlReplacementsFor(this.win);
+
     const platform = platformFor(this.win);
     /** @private @const {boolean} */
     this.isIosSafari_ = platform.isIos() && platform.isSafari();
 
+    /** @private {!Array<!function(!Event)|undefined>} */
+    this.boundHandlers_ = [];
+
     // Only intercept clicks when iframed.
     if (this.viewer_.isIframed() && this.viewer_.isOvertakeHistory()) {
-      /** @private @const {!function(!Event)|undefined} */
-      this.boundHandle_ = this.handle_.bind(this);
+      this.boundHandlers_.push(this.handle_.bind(this));
       this.win.document.documentElement.addEventListener(
-          'click', this.boundHandle_);
+          'click', this.boundHandlers_[this.boundHandlers_.length - 1]);
     }
+    // Add capture phase click handler for anchor target href expansion.
+    this.boundHandlers_.push(this.handle_.bind(this, true));
```

Comment:
It it missing the code that does the capture phase? Right now it looks like both will happen in the bubble phase.